src/TorchSharp/Utils/TensorAccessor.cs

```diff
@@ -231,8 +242,38 @@ private void validate(long index)
             if (index >= Count) throw new IndexOutOfRangeException();
         }
 
+        private void CopyContiguous(T[] array, int index=0, int count=0)
+        {
+             if (!_tensor.is_contiguous())
+                 throw new Exception("The tensor is not contiguous");
+             var shps = _tensor.shape;
+             long TempCount = 1;
+             for (int i = 0; i < shps.Length; i++)
+                 TempCount *= shps[i]; //Theorically the numel is simple as product of each element shape
```

Comment:
What do you mean 'theoretically' -- that's the definition of numel().

Reply:
My bad, my brain was in another place with another proyect.